ecs-agent/daemonimages/csidriver/util/fs/fs.go

```diff
@@ -0,0 +1,51 @@
+//go:build linux || darwin
+// +build linux darwin
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package fs
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+type UsageInfo struct {
```

Comment:
nit: since this struct is used by all three fs*.go files, perhaps this could be defined in a common struct file and have all three files import the struct

Reply:
unless this was taken from the existing EBS CSI driver then please make this as non-blocking

Reply:
The other two fs files are for different platforms and this `UsageInfo` in this fs file doesn't apply to all platforms.

Reply:
> unless this was taken from the existing EBS CSI driver then please make this as non-blocking

Yes, existing ebs csi is doing this same and definitely this won't be a blocker.
But I think your suggestion makes sense. I can try to make the change later.